mitosheet/src/components/elements/MultiToggleBox.tsx

```diff
@@ -260,7 +260,12 @@ const MultiToggleBox = (props: {
                             if (props.disabled) {
                                 return;
                             }
-                            toggleAllIndexes(nonDisabledDisplayedIndexes, !displayedNonDisabledAllToggled)
+
+                            if (!displayedNonDisabledAllToggled) {
```

Comment:
Is there a better name for this `displayedNonDisabledAllToggled`? It's quite hard to understand what this is doing. 

Or could you at least add a comment explaining the different parts. 

Displayed = not removed from the search
Non Disabled = not disabled 
All Toggled = The state of the toggle all button?

Reply:
gave it a much better name, it was confusing